lib/SimpleSAML/Utils/HTTP.php

```diff
@@ -85,6 +85,10 @@ private static function getServerHTTPS()
             return false;
         }
 
+        if ($_SERVER['HTTPS'] === null) {
```

Comment:
> It is explicitly defined to null, which is not something PHP is going to do itself, and therefore it must be someone, somewhere, doing it explicitly.
We use it together with shopware which is based on Zend Framework and maybe the framework set the HTTPS to null. I don`t know why. For us the simplest solution is to add this condition, because the key "HTTPS" exists but the value is null which means --> no HTTPS is activated.